sematic/ui/src/runs/RunIndex.tsx

```diff
@@ -46,37 +44,130 @@ export function RunRow(props: RunRowProps) {
         <Id id={run.id} trimTo={8} />
       </TableCell>
       <TableCell onClick={props.onClick}>
-        {props.noRunLink && run.name}
-        {!props.noRunLink && (
-          <Link href={"/runs/" + run.id} underline="hover">
-            {run.name}
-          </Link>
-        )}
+        <Typography variant="h6">
+          {props.noRunLink && run.name}
+          {!props.noRunLink && (
+            <Link href={`/pipelines/${run.calculator_path}/${run.id}`} underline="hover">
+              {run.name}
+            </Link>
+          )}
+        </Typography>
         {calculatorPath}
       </TableCell>
       <TableCell>
         <Tags tags={run.tags || []} />
       </TableCell>
       <TableCell onClick={props.onClick}>
-        <TimeAgo date={run.created_at} />
-        {createdAt}
+        <TimeAgo date={run.created_at} /><RunTime run={run} prefix="in" />
       </TableCell>
       <TableCell onClick={props.onClick}>
-        <RunStateChip run={run} />
+        <RunStateChip run={run} variant="full" />
       </TableCell>
     </TableRow>
   );
 }
 
+const StyledContainer = styled('div')`
+  height: 100%;
+  transform: translate(0);
+  width: fit-content;
+  margin: auto;
+`;
+
+const StyledScroller = styled(Container)`
+  padding-top: ${spacing(10)};
+  height: 100%;
+  overflow-y: scroll;
+  display: flex;
+  flex-direction: column;
+
+  @media (min-width: 1280px) {
+    min-width: 800px;
+  }
+
+  & .${buttonClasses.root} {
+    height: 100%;
+  }
+
+  & .${textFieldClasses.root} {
+    width: 100%;
+  }
+
+  & .search-bar {
+    padding-top: ${spacing(10)};
+    padding-bottom: ${spacing(10)};
+    display: flex;
+    flex-direction: row;
+
+    & > :first-child {
+      padding-right: ${spacing(10)};
+      flex-grow: 1
+    }
+  }
+
+  & form {
+    position: sticky;
+    top: -${spacing(15)};
+    background-color: ${({theme}) => theme.palette.background.paper};
+    z-index: 200;
```

Comment:
Is this so the form remains visible while you're scrolling down?

Reply:
Exactly.

Reply:
The layout strategy is changed. This is not applicable anymore.